### ISSUE TITLE
patches: openwrt/imagebuilder-custom-postinst-script: does not apply on recent OpenWrt-19.07

### ISSUE THREAD

**Issue by SvenRoederer** [freifunk-berlin/firmware#830]
I just tried to compile branch "master-monthly" (c8d5270286623df24bbe3178e0d55c6a6040cc93) and found it fails on patching.
```
--- Patching module 'openwrt' ---
Applying: tools/mkimake: fix handling of long pathnames
Applying: tools/u-boot: enlage max pathlength for mkimage
Applying: enable "sysupgrade -u" by default
Applying: kernel: avoid underscore in *6lowpan package names
Applying: mac80211_mesh_no_fwding_by_default
Applying: openwrt-release_use_configured_revision
Applying: hostapd_nolegacy_by_default
Applying: imagebuilder-custom-postinst-script
error: patch failed: target/imagebuilder/files/Makefile:105
error: target/imagebuilder/files/Makefile: patch does not apply
Patch failed at 0001 imagebuilder-custom-postinst-script
hint: Use 'git am --show-current-patch' to see the failed patch
```

So there must be something conflicting in 520581d11b6f118ebd49b8a1504063f7a3cface8.

**Comment by SvenRoederer** [comment]
To get such things detected automatically it would be really great to get a CI / CT working. It might be the buildbot or an alternative as discussed in #804 .

To ease updating the patches merging #794 would also much appreciated.

**Comment by SvenRoederer** [comment]
fixed by 835bcb9  & 9d62f88 (for daily-branch)
